cmd/ui/src/views/DatabaseManagement/DatabaseManagement.tsx

```diff
@@ -0,0 +1,307 @@
+            dispatch({ type: 'mutation_error' });
+        },
+        onSuccess: () => {
+            // show UI message that data deletion is happening
+            dispatch({ type: 'mutation_success' });
+        },
+    });
+
+    const handleMutation = () => {
+        mutation.mutate({
+            deleteThisData: {
+                collectedGraphData,
+                dataQualityHistory,
+                fileIngestHistory,
+                highValueSelectors,
+            },
+            assetGroupId: tierZeroAssetGroupId,
+        });
+    };
+
+    return { handleMutation, state, dispatch };
+};
+
+const DatabaseManagement = () => {
+    const theme = useTheme();
+
+    const { handleMutation, state, dispatch } = useDatabaseManagement();
+
+    const { collectedGraphData, highValueSelectors, fileIngestHistory, dataQualityHistory } = state;
+
+    const handleCheckbox = (event: React.ChangeEvent<HTMLInputElement>) => {
+        dispatch({
+            type: 'selection',
+            targetName: event.target.name,
+            checked: event.target.checked,
+        });
+    };
+
+    return (
+        <ContentPage title='Clear BloodHound data'>
+            <div>
+                <Typography variant='body1'>
+                    Manage your BloodHound data. Select from the options below which data should be deleted.
+                </Typography>
+                <Typography variant='body1'>
+                    <strong>Caution: </strong> This change is irreversible and will delete data from your environment.
+                </Typography>
+
+                <Box display='flex' flexDirection='column' alignItems='start'>
+                    <FormControl
+                        variant='standard'
+                        sx={{ paddingBlock: 2 }}
+                        error={state.noSelectionError || state.mutationError}>
+                        {state.noSelectionError ? (
+                            <Box color={theme.palette.error.main} display='flex' alignItems='center' gap='0.3rem'>
+                                <FontAwesomeIcon icon={faCircleXmark} />
+                                <FormHelperText sx={{ marginTop: '1.5px', color: theme.palette.error.main }}>
+                                    Please make a selection.
+                                </FormHelperText>
+                            </Box>
+                        ) : null}
+
+                        {state.mutationError ? (
+                            <Box color={theme.palette.error.main} display='flex' alignItems='center' gap='0.3rem'>
+                                <FontAwesomeIcon icon={faCircleXmark} />
+                                <FormHelperText sx={{ marginTop: '1.5px', color: theme.palette.error.main }}>
+                                    There was an error processing your request.
+                                </FormHelperText>
+                            </Box>
+                        ) : null}
```

Comment:
We use the MUI Alert components in other places in the app. This looks like a good place to use an <Alert/> so that these kinds of messages look consistent.